lightly/models/modules/heads.py

```diff
@@ -694,6 +694,38 @@ def __init__(
         )
 
 
+
+class VICRegProjectionHead(ProjectionHead):
```

Comment:
VICReg actually uses a two-layer projection head for experiments with 100 epochs of training.

Reply:
Thanks! Made an issue to also change it in the docs #1397